charts/sonarqube/templates/secret.yaml

```diff
@@ -29,6 +29,22 @@ data:
   SONAR_WEB_SYSTEMPASSCODE: {{ .Values.monitoringPasscode | b64enc | quote }}
 {{- end }}
 ---
+{{- if and .Values.monitoringPasscode .Values.prometheusMonitoring.podMonitor.enabled}}
+apiVersion: v1
```

Comment:
Is that Secret a duplicate of the one above ? on line 18 to 30 ? Apart from the confition it looks like the same, and both would be applied when `if and .Values.monitoringPasscode .Values.prometheusMonitoring.podMonitor.enabled` condition is true ?

Reply:
Unfortunately, we need the same secret both in the release and the podmonitor namespaces. But such duplicate code can be refactored. I've just defined [a task](https://sonarsource.atlassian.net/browse/SQHELM-117) for it.